packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -338,17 +338,20 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
           this._closeHttpSpan(span);
         });
         response.on('error', (error: Err) => {
+          diag.debug('outgoingRequest on error()');
```

Comment:
it can be `outgoingRequest on response error()` but then we probably need to change this line https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-instrumentation-http/src/http.ts#L314